docker-compose.yml

```diff
@@ -21,41 +21,48 @@ services:
       start_period: 10s
   rabbitmq:
     image: "rabbitmq:3-management"
-    hostname: "rabbitmq"
+    hostname: "rabbitmq-go-quickstart"
     ports:
       - "5673:5672" # RabbitMQ
       - "15673:15672" # Management UI
     environment:
       RABBITMQ_DEFAULT_USER: "user"
       RABBITMQ_DEFAULT_PASS: "password"
     volumes:
-      - "hatchet_rabbitmq_data:/var/lib/rabbitmq"
-      - "hatchet_rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf" # Configuration file mount
+      - "hatchet_quickstart_rabbitmq_data:/var/lib/rabbitmq"
+      - "hatchet_quickstart_rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf" # Configuration file mount
     healthcheck:
       test: ["CMD", "rabbitmqctl", "status"]
       interval: 10s
       timeout: 10s
       retries: 5
   migration:
-    image: ghcr.io/hatchet-dev/hatchet/hatchet-migrate:v0.8.0
+    image: ghcr.io/hatchet-dev/hatchet/hatchet-migrate:v0.28.2
```

Comment:
automation is planned and upcoming 😁 I think that will be nice because we will see if anything fails, and the main branch will always work even if an update would break things